src/server/gc_worker/gc_manager.rs

```diff
@@ -17,7 +17,7 @@ use super::config::GcWorkerConfigManager;
 use super::gc_worker::{sync_gc, GcSafePointProvider, GcTask};
 use super::{is_compaction_filter_allowd, Result};
 
-const POLL_SAFE_POINT_INTERVAL_SECS: u64 = 60;
+const POLL_SAFE_POINT_INTERVAL_SECS: u64 = 10;
```

Comment:
Just curious, why we need to change the safe point fetching interval? And there should be much overhead added for changing this?

Reply:
It can make GC more timely.